Makefile

```diff
@@ -1,6 +1,8 @@
 # Variables. Yes.
 DOCKER=docker
 DOCKER_BUILDKIT=0
+IMAGE_NAME=rwint9-site
```

Comment:
A good time to rename the image! Could be just `rwint-site` to match the value in `local/.env.example` (and time to rename the repo too, but that's probably worth its own ticket.)

Reply:
Yes, I'd also like to rename it but also already have a `rwint-site` repo and that will probably imply several changes on the config used on the server. Let's make an OPS ticket and check with Cafuego.